openbas-front/src/admin/components/common/injects/expectations/ExpectationPopover.tsx

```diff
@@ -29,12 +29,20 @@ const ExpectationPopover: FunctionComponent<ExpectationPopoverProps> = ({
   const [openDelete, setOpenDelete] = useState(false);
   const [openEdit, setOpenEdit] = useState(false);
 
+  const getExpirationTime = (expirationTime: number): number => {
+    if (expirationTime !== null || expirationTime !== undefined) {
+      return expirationTime;
+    }
+    return (expectation.expectation_type === 'DETECTION' || expectation.expectation_type === 'PREVENTION') ? 21600 : 3600;
+  };
```

Comment:
Maybe create a const for this number: const EXPIRATION_TIME_SIX_HOURS = 21600 (avoid magic numbers)